src/core/src/local_loggers/CompositeLogger.py

```diff
@@ -36,7 +38,7 @@ def log(message):
         """log output"""
         message = CompositeLogger.__remove_substring_from_message(message, Constants.ERROR_ADDED_TO_STATUS)
         for line in message.splitlines():  # allows the extended file logger to strip unnecessary white space
-            print(line)
+            sys.stdout.write(line)
```

Comment:
As noted in my last comment on this, print automatically inserts a new line character at the end of the line. It also ensures that whatever is passed in is converted to string if required. Sys.stdout.write doesn't. Please ensure the change is equivalent or optionally revert to print.